cypress/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM cypress/included:5.2.0
+
+WORKDIR /root
+
+RUN apt-get install curl
+
+ENV CYPRESS_VIDEO=false
```

Comment:
you can pass this in as a -e option when doing `docker run` if need be. this is optional.

Reply:
Indeed.  This came across with Vega's dockerfile. I guess its a sensible default as the only time we want the video is when running locally.